client/my-sites/stats/activity-log/index.jsx

```diff
@@ -353,16 +353,19 @@ class ActivityLog extends Component {
 		);
 
 		const timePeriod = ( () => {
+			const today = this.applySiteOffset( moment.utc( Date.now() ) );
 			let last = null;
 
 			return ( { rewindId } ) => {
-				const ts = 1000 * rewindId;
+				const ts = this.applySiteOffset( moment.utc( rewindId * 1000 ) );
 
-				if ( null === last || moment( ts ).format( 'D' ) !== moment( last ).format( 'D' ) ) {
+				if ( null === last || ! ts.isSame( last, 'day' ) ) {
 					last = ts;
 					return (
 						<h2 className="activity-log__time-period" key={ `time-period-${ ts }` }>
-							{ moment( ts ).format( 'LL' ) }
+							{ ts.isSame( today, 'day' )
+								? ts.format( translate( 'LL[ — Today]', { context: 'moment format string' } ) )
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.

Reply:
Yes, this really needs some clarification. I am reviewing translations for that string, and really not clear what here can be translated, and what not. How is that meant to be used?

Reply:
thanks for questioning it @zinigor - maybe you can help me write my note to the translators

first of all, it requires knowing how to write `moment` format strings.

the intent is to show a short date string followed by ` - Today`, but I acknowledge that ` - Today` may not make sense in different locales. that is, we want to show that a date string is also today but only in a way that makes sense for the locale. previously we were splitting the string in two and that would end up I think even more awkward to translate